Static/Row.swift

```diff
@@ -142,6 +153,8 @@ public struct Row: Hashable, Equatable {
         self.image = image
         self.accessory = accessory
         self.cellClass = cellClass ?? Value1Cell.self
+        self.cellNib = cellNib ?? nil
```

Comment:
We can remove the `?? nil` because the initializer specifies a default value for it. :sparkles: